packages/features/settings/BookerLayoutSelector.tsx

```diff
@@ -85,19 +85,28 @@ const BookerLayoutFields = ({ settings, onChange, showUserSettings }: BookerLayo
   // Converts the settings array into a boolean object, which can be used as form values.
   const toggleValues: BookerLayoutState = bookerLayoutOptions.reduce((layouts, layout) => {
     layouts[layout] = !shownSettings?.enabledLayouts
-      ? true
```

Comment:
Before if no settings were set, all layouts would be enabled. Now we only enable the layouts that are enabled in the default settings (so only month view as of now).